### PR TITLE
Fix HexEdit cursor alignment

### PR DESCRIPTION
**Pull request by mkow**
Improved hex cursor alignment (it wasn't properly centered in both ASCII and hex view).

### CODE REVIEW

**Comment by adamiwaniuk** [comment]
In my opinion in hex view previous version was better

**Comment by mkow** [comment]
Try now. There was a problem with `spaceAfterByte_ / 2` (its parity depends on DPI).

File: src/ui/hexedit.cc [hunk 1]
```diff
@@ -409,17 +411,47 @@ QRect HexEdit::bytePosToRect(qint64 pos, bool ascii, qint64 char_pos) {
...
-  return QRect(xPos - spaceAfterByte_ / 2,
+  return QRect(xPos - hex_border_spacing_x,
+               yPos - charHeight_ + verticalByteBorderMargin_, width,
+               charHeight_);
+}
+
+// Returns where a border of a given byte should be drawn. Borders can overlap
+// with each other, because of centering on different DPIs.
+QRect HexEdit::bytePosToBorder(qint64 pos, bool ascii, qint64 char_pos) {
+  qint64 columnNum = pos % bytesPerRow_;
```
Comment (adamiwaniuk) [inline src/ui/hexedit.cc]: column_num and other vars in this function

**Comment by mkow** [comment]
Another approach to fix this in https://github.com/codilime/veles/pull/252

**Pull request closed without merging**